internal/pkg/scaffold/ansible/go_mod.go

```diff
@@ -0,0 +1,124 @@
+// Copyright 2019 The Operator-SDK Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ansible
+
+import (
+	"fmt"
+
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/input"
+	"github.com/operator-framework/operator-sdk/internal/pkg/scaffold/internal/deps"
+)
+
+const GoModFile = "go.mod"
+
+// GoMod - the go.mod file for an Ansible hybrid operator.
+type GoMod struct {
+	input.Input
+}
+
+func (s *GoMod) GetInput() (input.Input, error) {
+	if s.Path == "" {
+		s.Path = GoModFile
+	}
+	s.TemplateBody = goModTmpl
+	return s.Input, nil
+}
+
+const goModTmpl = `module {{ .Repo }}
+
+require (
+	cloud.google.com/go v0.37.2 // indirect
```

Comment:
Just a suggestion: we have some common deps here, could we extract them, so when we need to update them we would just need to bump it in one place?

Not a blocker and can be done in a follow up!

Reply:
I like the idea. We can follow up on this in another PR.